test/runtime/runtime-kotlin.ts

```diff
@@ -0,0 +1,28 @@
+import * as fs from 'fs';
```

Comment:
This whole file needs to follow https://github.com/asyncapi/modelina/blob/master/test/runtime/runtime-java.ts

Just for kotlin instead 🙂